docs/getting_started/install.md

```diff
@@ -124,6 +124,7 @@ your project.
     {
         public string $fromEmail  = 'your_mail@example.com';
         public string $fromName   = 'your name';
+        public string $mailType = 'html';
```

Comment:
This is the Config file to the CI4 application.
It is not sure that the dev uses HTML for all emails that s/he sends.

But Shield uses HTML mails, so should Shield send the mails as `'html'` at the runtime?

Reply:
I sent #1020